staging/src/k8s.io/apiextensions-apiserver/pkg/controller/openapi/builder.go

```diff
@@ -96,33 +96,39 @@ func BuildSwagger(crd *apiextensions.CustomResourceDefinition, version string) (
 
 	routes := make([]*restful.RouteBuilder, 0)
 	root := fmt.Sprintf("/apis/%s/%s/%s", b.group, b.version, b.plural)
+
+	var params []*restful.Parameter
+	nameParam := b.ws.PathParameter("name", "name of the "+b.kind).DataType("string")
+	namespaceParam := b.ws.PathParameter("namespace", "object name and auth scope, such as for teams and projects").DataType("string")
```

Comment:
move down to line `105` - could be inside `if b.namespaced`

Reply:
done